StreamVideo.xcodeproj/project.pbxproj

```diff
@@ -3289,9 +3289,10 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 15;
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
why is this moved back to 1?

Reply:
It always should be one as it's fully managed by fastlane. Changing it manually will lead to issues in the TestFlight build.